root/static/styles/extra/homepage.less

```diff
@@ -11,6 +11,39 @@
     }
 }
 
+#app-download {
+    .section {
+        padding: 30px 0;
+        overflow: hidden;
```

Comment:
Is this rule needed?  I'm skeptical of it because I'd think we should normally be trying to resize content or let it wrap naturally instead of hiding it. :)  But maybe it serves some purpose I'm missing.

Reply:
Fixed